scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -671,7 +664,8 @@ private ConflictSolverResult solveActivityNetworkConflict(
                                                                                       1,
                                                                                       Optional.empty(),
                                                                                       Optional.of(true),
-                                                                                      Optional.empty());
+                                                                                      Optional.empty(),
+                                                                                      ScheduleAt.LATEST);
```

Comment:
I think that `ScheduleAt.LATEST` should be chosen by Recurrence Goal, and not be mandatory for all network conflicts.